pkg/virt-api/webhooks/validating-webhook/admitters/vmexport-admitter.go

```diff
@@ -132,44 +128,86 @@ func (admitter *VMExportAdmitter) Admit(ar *admissionv1.AdmissionReview) *admiss
 	return &reviewResponse
 }
 
-func (admitter *VMExportAdmitter) validatePVC(field *k8sfield.Path, namespace, name string) ([]metav1.StatusCause, error) {
+func (admitter *VMExportAdmitter) validatePVCName(field *k8sfield.Path, name string) []metav1.StatusCause {
 	if name == "" {
 		return []metav1.StatusCause{
 			{
 				Type:    metav1.CauseTypeFieldValueInvalid,
 				Message: "PVC name must not be empty",
 				Field:   field.String(),
 			},
-		}, nil
+		}
+	}
+
+	return []metav1.StatusCause{}
+}
+
+func (admitter *VMExportAdmitter) validatePVCApiGroup(field *k8sfield.Path, apigroup *string) []metav1.StatusCause {
+	if apigroup != nil && *apigroup != "" {
```

Comment:
My bad, I have looked at your push history right now. So maybe it is the `Message` that is wrong

Reply:
Could you also specify why `export.kubevirt.io` is not allowed? Thanks

Reply:
So we are checking that the LocalObjectReference we pass is one of the allowed values. There are 3 possible values right now:
`pvc`, `vmsnapshot`, `vm`. PVC does not have an APIGroup (aka it can be omitted or blank), VMSnapshot has an APIGroup of `snapshot.kubevirt.io` and VM has an APIGroup `kubevirt.io`

Reply:
Right, I got confused! Thanks for the explanation!